compiler-core/src/error.rs

```diff
@@ -556,6 +560,24 @@ to `src/{module}.gleam`"
                 location: None,
             },
 
+            Error::OutputFilesAlreadyExist { file_names } => Diagnostic {
+                title: "File(s) already exist in target directory".into(),
```

Comment:
When there's only 1 file this should say "File", and when there's another number it should say "Files"